javascript/npm/npm-features-ii/passing-through-command-line-arguments-in-npm-scripts.md

```diff
@@ -89,3 +88,23 @@ npm run myScript \
 - argvs
 - arg
 - `process.argv[0]`
+
+---
+
+## Footnotes
+
+[1:Calling a scrip directly]
+
+When we run scripts through `npm`, we're effectively using `npm` as a middleman.
+
+This means that passing regular arguments would be passing inputs to the `npm` program **directly**, not the script that it runs **underneath**.
+
+```bash
+# passing inputs to npm directly
+npm run someCommand --arg=value
+
+# passing inputs to the someCommand script
+npm run someCommand -- --arg=value
+```
+
+The `--` let's us skip `npm` and pass arguments into the script underneath `npm`.
```

Comment:
Quick paraphrase because we used "underneath" before

```suggestion
The `--` let's us skip `npm` and pass arguments into the script that `npm` is calling.
```